v-next/hardhat-utils/src/eth.ts

```diff
@@ -1,19 +1,61 @@
 import type { PrefixedHexString } from "./hex.js";
```

Comment:
This module is starting to have many unrelated things. Maybe we should split it into multiple ones.

Reply:
I've removed the library linking logic from this PR and will submit it in a follow-up PR.